### PR TITLE
git graduation yearbook

### PR DESCRIPTION
**Pull request by pratapnea**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/pratapnea/pratapnea.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pratap Neaupane # No longer than 18 characters
+institution: Nepathya College 🚩 # no longer than 58 characters
+profile_pic: pratapnea.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: "Everyone has thier own role in someone's life and some people are really great to meet. Thankyou everyone!" # no longer than 100 characters
```
Comment (Mohammed-Shoaib) [inline _data/pratapnea/pratapnea.md:5]: Must **not** be longer than 100 characters. Yours is **108** characters.

Typos present:
* thier → their
* Thankyou → Thank you

File: _data/pratapnea/pratapnea.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pratap Neaupane # No longer than 18 characters
+institution: Nepathya College 🚩 # no longer than 58 characters
```
Comment (Mohammed-Shoaib) [inline _data/pratapnea/pratapnea.md:3]: Remove the 🚩(flag) as it is a non-ascii character.

**Comment by pratapnea** [comment]
Thank you i have made changes

**Comment by pratapnea** [comment]
Thank you there. I have made all the necessary changes. Please look into it
again.

Regard,
Pratap

File: _data/pratapnea/pratapnea.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pratap Neaupane # No longer than 18 characters
+institution: Nepathya College # no longer than 58 characters
+profile_pic: pratapnea.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: "Everyone has their own role in someone's life and its really great to meet them. Thank you everyone!" # no longer than 100 characters
```
Comment (Mohammed-Shoaib) [inline _data/pratapnea/pratapnea.md:5]: Quote must **not** be longer than 100 characters, yours is **102** characters with the double-quotes. You can choose to remove the double-quotes or make your quote smaller.

**Comment by pratapnea** [comment]
Thakyou there,
I have made my quote shorter and removed the double-quote. Please have a
look and let me know.

Regards,
Pratap

**Pull request merged**